### DISCUSSION TITLE
What is the stable version to install with azure container support

### DISCUSSION THREAD

**Discussion by srajasimman** [Sunbird-cQube/community#28] [Q&A] [answered]
Hi, @jai-ekstep,

Please let us know what is the production-ready stable version to install with azure container support.

**Comment by SreenivasNimmagadda** [comment, marked as answer]
Hi Rajasimman, 

As of now cQube is supporting the AWS as well as Private Data Center. 
The current stable version for AWS  & PDC is cQube v3.3

For the Azure, Still we haven't rolled out the stable version.
